src/dll/Hooks/ExecuteProcess.cpp

```diff
@@ -21,11 +21,12 @@ bool _Global_ExecuteProcess(void* a1, RED4ext::CString& aCommand, FixedWString&
         return Global_ExecuteProcess(a1, aCommand, aArgs, aCurrentDirectory, a5);
     }
 
-    auto scriptCompilationSystem = App::Get()->GetScriptCompilationSystem();
+    auto str = App::Get()->GetScriptCompilationSystem()->GetCompilationArgs(aArgs);
     
     FixedWString newArgs;
-    newArgs.str = scriptCompilationSystem->GetCompilationArgs(aArgs).c_str();
+    newArgs.str = str.c_str();
     newArgs.length = newArgs.maxLength = wcslen(newArgs.str);
```

Comment:
I would say:
```suggestion
    newArgs.length = str.length();
    newArgs.maxLength = str.capacity();
```
I noticed now that they are on the same line, don't really like multiple assignment on the same line.